Source/WebCore/rendering/svg/legacy/LegacyRenderSVGShape.h

```diff
@@ -123,11 +125,14 @@ class LegacyRenderSVGShape : public LegacyRenderSVGModelObject {
     bool nodeAtFloatPoint(const HitTestRequest&, HitTestResult&, const FloatPoint& pointInParent, HitTestAction) final;
 
     FloatRect objectBoundingBox() const final { return m_fillBoundingBox; }
-    FloatRect strokeBoundingBox() const final;
     FloatRect calculateStrokeBoundingBox() const;
     void updateRepaintBoundingBox();
 
     bool setupNonScalingStrokeContext(AffineTransform&, GraphicsContextStateSaver&);
+
+    FloatRect calculateApproximateStrokeBoundingBox() const;
+    FloatRect calculateApproximateScalingStrokeBoundingBox(FloatRect fillBoundingBox) const;
+    FloatRect calculateApproximateNonScalingStrokeBoundingBox(FloatRect fillBoundingBox) const;
```

Comment:
Please consider putting these two functions  (and maybe some of the guts of `calculateApproximateStrokeBoundingBox`?) in `SVGRenderSupport`, which is where we have some other code that is common to LBSE and Legacy.

Reply:
Moved!